src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsSplit.js

```diff
@@ -198,6 +198,11 @@ class IOUParticipantsSplit extends Component {
     render() {
         const maxParticipantsReached = this.props.participants.length === CONST.REPORT.MAXIMUM_PARTICIPANTS;
         const sections = this.getSections(maxParticipantsReached);
+        const headerMessage = maxParticipantsReached ? '' : getHeaderMessage(
```

Comment:
it is not readable. I would suggest swapping the condition so that condition would be `!maxParticipantsReached`.

Reply:
Done